st2api/Makefile

```diff
@@ -59,7 +59,7 @@ all: install
 
 install: wheelhouse changelog
 
-post_install:
+post_install: bdist_wheel
```

Comment:
We install pip wheels locally, not because of pip itself. But because of this
https://github.com/dennybaa/st2-packages/blob/st2_bundle/sources/st2actions/debian/rules#L35

In other words, there's bundle requiremets.txt which contain st2common, st2actions since pip can not fetch or wheels from pypi, we use local wheelhouse which is on line #35.